src/views/partials/av.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="av">
+  <!-- My-video (front)
```

Comment:
Let's remove all the comments from our views, since it gets pushed with the rest of the code to the DOM. Our comments shouldn't really be visible when someone inspects an element